x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_table_filters/rules_table_filters.tsx

```diff
@@ -101,6 +111,26 @@ const RulesTableFiltersComponent = () => {
           </EuiFilterButton>
         </EuiFilterGroup>
       </EuiFlexItem>
+
+      <EuiFlexItem grow={false}>
+        <EuiFilterGroup>
+          <EuiFilterButton
+            hasActiveFilters={enabled === true}
+            onClick={handleShowEnabledRulesClick}
+            data-test-subj="showEnabledRulesFilterButton"
+            withNext
+          >
+            {i18n.ENABLED_RULES}
+          </EuiFilterButton>
+          <EuiFilterButton
+            hasActiveFilters={enabled === false}
+            onClick={handleShowDisabledRulesClick}
+            data-test-subj="showDisabledRulesFilterButton"
+          >
+            {i18n.DISABLED_RULES}
+          </EuiFilterButton>
```

Comment:
Since space is of concern with regards to shrinking the search bar, we could always use a drop down for `Rule state` like they're doing on the Stack Mgmt Rules table (though w/o the extra width 😅):

<img width="386" alt="image" src="https://user-images.githubusercontent.com/2946766/216461356-7d03ae9e-1dbe-48cc-ab48-95c0221a1daa.png">

Reply:
We will need something like that for 8.8 since we plan to add even more filters. For now I'd like to keep it simple.